README.md

```diff
@@ -62,6 +62,7 @@ easy tool to use, and is targeted at non-developers.
 * [pgrok](https://www.proxy.jetzt/) [![pgrok github stars badge](https://img.shields.io/github/stars/jerson/pgrok?style=flat)](https://github.com/jerson/pgrok/stargazers) - Fork of ngrok 1.0, with more recent commits.
 * [SirTunnel](https://github.com/anderspitman/SirTunnel) [![SirTunnel github stars badge](https://img.shields.io/github/stars/anderspitman/SirTunnel?style=flat)](https://github.com/anderspitman/SirTunnel/stargazers) - Minimal, self-hosted, 0-config alternative to ngrok. Similar to sish but leverages Caddy+OpenSSH rather than custom server code.
 * [docker-tunnel](https://github.com/vitobotta/docker-tunnel) [![docker-tunnel github stars badge](https://img.shields.io/github/stars/vitobotta/docker-tunnel?style=flat)](https://github.com/vitobotta/docker-tunnel/stargazers) - Simple Docker-based nginx+SSH solution.
+* [OpenZiti](https://openziti.github.io) - [![OpenZiti github stars badge](https://img.shields.io/github/stars/openziti/ziti?style=flat)](https://github.com/openziti/ziti/stargazers) - OpenZiti is a fully open source zero trust overlay network (licensed Apache v2). The goal of OpenZiti is to extend zero trust all the way into your application, not just to your network. Consisting a fully featured mesh network, mTLS between all nodes, e2e encryption of tunnels, continous posture checking, client agents to support brownfield applications (which also support truly *private* DNS), SDKs for many laguages (and C SDK for FFI). It's a fully-featured tunneling solution.
```

Comment:
The sentence starting with "Consisting" has a grammer/syntax issue. I think it should be "Consisting **of** a" instead of "Consisting **a**"

Reply:
fixed. thx